pysph/cpy/tests/test_translator.py

```diff
@@ -1178,8 +1178,8 @@ def f(s_idx, s_p, d_idx, d_p, J=0, t=0.0, l=[0,0], xx=(0, 0)):
 
     # Then
     expect = dedent('''
-void f(long s_idx, __global double* s_p, long d_idx, __global int* d_p, long
-    J, double t, double* l, double* xx)
+WITHIN_KERNEL void f(long s_idx, __global double* s_p, long d_idx, __global
```

Comment:
Should we also change the `__global` to `GLOBAL_MEM` in the translator or will you do that in the next round of changes?